project/.gitattributes

```diff
@@ -0,0 +1,6 @@
+.editorconfig    export-ignore
+.gitattributes   export-ignore
+.gitignore       export-ignore
+CONTRIBUTING.md  export-ignore
+
+/Resources/doc   export-ignore
```

Comment:
This is not always this path. Pleas use `{{ docs_path }}` instead.

Example: https://github.com/sonata-project/dev-kit/blob/ae18f3ea3755f543d83b4f713b752dc66398216d/project/Makefile#L17